### ISSUE TITLE
Admin tries to load assets from localhost after running twill-dev

### ISSUE THREAD

**Issue by zipavlin** [area17/twill#152]
Admin assets try to load from localhost:8080 after running ```npm run twill-dev```:
```
http://localhost:8080//assets/admin/css/app.css
http://localhost:8080//assets/admin/js/manifest.js
http://localhost:8080//assets/admin/js/vendor.js
http://localhost:8080//assets/admin/js/main-form.js
```

This of course works as expected when ```twill-dev``` command is running, but assets should probably be requested from ```<domain>/assets/admin/*``` when we are no longer watching for changes - this results in assets not being found and admin being not-really-useful.

Running ```npm run twill-build``` doesn't change this.

I have no idea, what is causing this. I am on Win 10, running Laravel app from homestead.
Any help is appreciated! :)

**Comment by m4n1ok** [comment]
Hi @zipavlin, 

Did you try to remove hot file in public directory ? https://github.com/area17/twill/issues/49#issuecomment-433123633

Hope that helps.

**Comment by ifox** [comment]
Hi @zipavlin,

thanks @m4n1ok! I wish Mix would automatically delete the hot file when stopping the HMR server but I guess a solution would be to add `rm -f public/hot` to the start of the `twill-build` script.

**Comment by zipavlin** [comment]
Thank you @m4n1ok and @ifox.
Deleting ```public/hot``` and running ```npm run twill-build``` worked!

**Comment by raymondtri** [comment]
I had a similar issue with this where I was getting an invalid host/origin error while reloading hot.

npm run twill-dev still didn't work for me and it took quite some time to figure out.

The fix was modifying the actual vendor/area17/twill/package.json to include the configuration option "--disable-host-check" within the hot script. The security vulnerability shouldn't be important because you wouldn't be using a hot reload script in a production environment. The reason I had to modify the twill package.json is because that seemed to be what is being run concurrently in the twill-dev script.

**Comment by ifox** [comment]
Hey @raymondtri, I went through the same issue a couple of days ago and came up with the exact same conclusion, this is perfectly fine for local development. We should be able to release 1.2.2 this week with that fix included.

Reference: https://github.com/JeffreyWay/laravel-mix/issues/1955

**Comment by raymondtri** [comment]
Awesome!

Is there support for rendering custom vue components deeper than what is generated by blocks? 

When we are utilizing preview mechanisms in the backend, it would be essential to render custom vue components as well as blocks. This would drastically expand the ability to extend twill. Not to mention if you are using twill for frontend, it would be fantastic to be able to include custom vue components which could be called in frontend blades as well. I spent a few minutes briefly messing around with "twill-copy-vue" and "twill-clean-vue" methods but all I succeeded in doing was showing permanent load screens in my admin backend, ha.
